tests/openvino/test_modeling.py

```diff
@@ -749,6 +748,11 @@ def test_compare_to_transformers(self, model_arch):
         )
 
         ov_outputs = ov_model.generate(**tokens, generation_config=gen_config)
+
+        # TODO: update _update_model_kwargs_for_generation so that it's compatibile with transformers >= v4.42.0
+        if model_arch not in ["chatglm", "glm4"] and is_transformers_version(">=", "4.42.0"):
```

Comment:
@eaidova  `_update_model_kwargs_for_generation` needs to be updated to enable generation with transformers v4.42.0 https://huggingface.co/katuni4ka/tiny-random-chatglm2/blob/main/modeling_chatglm.py#L871 as now `_extract_past_from_model_output` also returns `cache_name`

https://github.com/huggingface/transformers/blob/82486e5995ed0a65520b10ce1ea938214a199231/src/transformers/generation/utils.py#L634

Reply:
ok, I'll update

Reply:
thnaks a lot @eaidova

Reply:
https://huggingface.co/katuni4ka/tiny-random-chatglm2/commit/71090c68d20238495ebf43f6c2f9ae194b1a7a2d